### ISSUE TITLE
Custom view padding is not taken into account, causing it to get cut off

### ISSUE THREAD

**Issue by guahirah** [skydoves/Balloon#416]
**Please complete the following information:**
- Library Version [e.g. v1.5..2]
- Affected Device(s) [All]
 
If we use a custom view with padding and fixed width for ballon, the padding is not taken into account, causing it to get cut off.

custom_view.xml
```
<androidx.constraintlayout.widget.ConstraintLayout
        android:layout_width="match_parent"
        android:layout_height="wrap_content"
        android:padding="10dp"> // set the padding

        <TextView
            android:layout_width="wrap_content"
            android:layout_height="wrap_content"
            app:layout_constraintStart_toStartOf="parent"
            app:layout_constraintTop_toTopOf="parent" />
</androidx.constraintlayout.widget.ConstraintLayout>
```

Ballon initialize
```
balloon = createBalloon(context) {
                setArrowSize(0)
                setLayout(customView) // custom view
                setWidthRatio(widthRatio) // fixed width
                setIsVisibleArrow(false)
                setBalloonAnimation(BalloonAnimation.FADE)
            }
```

**Comment by skydoves** [comment]
Hey @guahirah, you're using the `ConstraintLayout` wrong. You should update the `TextView` like the below:

```kotlin
<TextView
            android:layout_width="0dp"
            android:layout_height="0dp"
            app:layout_constraintTop_toTopOf="parent"
            app:layout_constraintStart_toStartOf="parent"
            app:layout_constraintEnd_toEndOf="parent" />
</androidx.constraintlayout.widget.ConstraintLayout>
```

Thanks!

**Comment by guahirah** [comment]
Hi, @skydoves
I have tested your suggestion, but the content is still getting cut off. It will be fixed if I remove the ConstraintLayout's padding and set it to Textview instead. But if we have a complex layout, it gets complicated.

Thank you in advance.

**Comment by guahirah** [comment]
Hi, @skydoves
I've tested your suggestion, but it leads to another problem. It will have a extra space at the bottom of the window.

This will be fixed if we remove the ConstraintLayout (Root) padding and set it to Textview instead. But if we have a complex layout, it gets complicated.

Thank you in advance.

**Comment by skydoves** [comment]
Hey @guahirah, would you please test with the [sample project](https://github.com/skydoves/Balloon/blob/main/app/src/main/res/layout/layout_custom_profile.xml)? You can give additional paddings on the layout file.

**Comment by guahirah** [comment]
I have tested with a sample project. Change the `ConstraintLayout` width to `match_parent`, and use fixed width on balloon initialization. It will have extra space at the bottom of the window.

`layout_custom_profile.xml` (Change width to `match_parent`)
```
<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
  xmlns:app="http://schemas.android.com/apk/res-auto"
  xmlns:tools="http://schemas.android.com/tools"
  android:layout_width="match_parent"
  android:layout_height="wrap_content"
  android:gravity="center"
  android:orientation="vertical"
  tools:background="@color/background800">
```

Ballon initialization (Use fixed width,  `setWidthRatio(0.5f)`)
```
return createBalloon(context) {
      //setWidth(BalloonSizeSpec.WRAP)
      setWidthRatio(0.5f)
      setHeight(BalloonSizeSpec.WRAP)
      setLayout(R.layout.layout_custom_profile)
      setArrowSize(10)
      setPadding(12)
      setArrowOrientation(ArrowOrientation.TOP)
      setArrowPositionRules(ArrowPositionRules.ALIGN_ANCHOR)
      setArrowPosition(0.5f)
      setCornerRadius(6f)
      setElevation(6)
      setBackgroundDrawableResource(R.drawable.background_gradient)
      setArrowColorMatchBalloon(true)
      setBalloonAnimation(BalloonAnimation.CIRCULAR)
      setDismissWhenTouchOutside(true)
      setDismissWhenShowAgain(true)
      setDismissWhenLifecycleOnPause(true)
      setLifecycleOwner(lifecycle)
    }
```

Result
![Screenshot_20230129_115131](https://user-images.githubusercontent.com/122971420/215305667-2c50ef20-da3b-404d-8357-f82808488cbb.png)
